code/headers/usart_connection.hpp

```diff
@@ -0,0 +1,53 @@
+/// @file
+/// @brief      base class for a usart implementation hardware, software and mock
```

Comment:
is this doxygen for the file or the class?

Reply:
file